ext/hook/uhook_attributes.stub.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/** @generate-class-entries */
+
+namespace DDTrace;
+
+require "Zend/zend_attributes.stub.php";
+
+#[\Attribute(\Attribute::TARGET_FUNCTION | \Attribute::TARGET_METHOD)]
+final class Traced {
```

Comment:
Other languages implemeting this (at least Java and .NET, haven't checked the others) use `Trace`. We could probably mimick that

Reply:
Would be fine by me too.